src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -221,20 +225,68 @@ def logs(
     num_lines: int = typer.Option(
         500, "--num-lines", help="Number of lines to retrieve."
     ),
+    previous_logs: bool = typer.Option(
+        False, "--previous-logs", "-p", help="Retrieve previous logs", is_flag=True
+    ),
+    since_timestamp: Optional[str] = typer.Option(
+        "", "--since", help="Timestamp to retrieve logs from"
+    ),
+    include_timestamps: bool = typer.Option(
+        False, "--include-timestamps", help="Include timestamps in logs", is_flag=True
+    ),
+    follow: bool = typer.Option(
+        False, "--follow", "-f", help="Continue polling for logs.", is_flag=True
+    ),
+    follow_interval: int = typer.Option(
+        2,
+        "--follow-interval",
+        help="Polling interval in seconds when using the --follow flag",
+    ),
```

Comment:
If we decide to have this functionality I would suggest combining this in into `--refresh-interval` that if provided will cause logs to be refreshed every few seconds. This also requires users to provide a number.